README.md

```diff
@@ -1,2 +1,65 @@
 # seismograph
+
 Utilities for gathering traction metrics around open source projects
+
+## Installation
+
+Optionally, create and active a virtual python environment.
+
+```nohighlight
+$ virtualenv venv
+$ source venv/bin/activate
+```
+
+Install required python libraries:
+
+```nohighlight
+$ pip install -r requirements.txt
+```
+
+## Configuration
+
+Copy the provided `config.yaml.dist` to `config.yaml` and edit according to your needs.
+
+- `global.github_api_token`: Optionally set the value of this key to a valid Github API token. Without a token, you might run into rate limits when using Seismograph frequently.
+- `global.projects`: For each software project you want to track, add one key here. Name this key however you want, but make sure it's unique within this file.
```

Comment:
All your examples have the `projects` at the top of the file, not inside `global`